composeml/label_maker.py

```diff
@@ -93,30 +92,26 @@ def df_to_labels(df):
             df = df.loc[df.index.notnull()]
             df.sort_index(inplace=True)
 
-            if df.empty:
-                return labels
-
             cutoff_time = offset_time(df.index, minimum_data)
 
             for example in range(num_examples_per_instance):
-                df = df[cutoff_time:]
-
-                if df.empty:
-                    skipped_iterations = num_examples_per_instance - example
-                    progress_bar.update(n=skipped_iterations)
+                if cutoff_time is None:
```

Comment:
why did this conditional change?

Reply:
It is dependent on the previous condition (e.g. if the data frame is empty then the cutoff time is already `None`). Also, removes the dependency on running this line beforehand.
```
df = df[cutoff_time:]
```